include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -193,15 +193,15 @@ struct __parallel_transform_reduce_device_kernel_submitter<__work_group_size, __
 // Submits the second kernel of the parallel_transform_reduce for mid-sized arrays.
 // Uses a single work groups to reduce __n preliminary results stored in __temp and returns a future object with the
 // result buffer.
-template <::std::uint16_t __work_group_size, ::std::uint8_t __iters_per_work_item, typename _Tp, typename _KernelName>
+template <typename _Tp, ::std::uint16_t __work_group_size, ::std::uint8_t __iters_per_work_itemx, typename _KernelName>
```

Comment:
I think `__iters_per_work_itemx` is a typo here.

Reply:
Good catch, thanks.